code/modules/projectiles/guns/energy/stun.dm

```diff
@@ -41,7 +41,17 @@
 	projectile_type = /obj/item/projectile/energy/electrode
 
 /obj/item/gun/energy/stunrevolver/moebius
-	name = "Moebius SP \"Suez\""	//Ersatz name 
+	name = "Moebius SP \"Suez\""	//Ersatz name
 	desc = "Also know as stunrevolver. A Moebius copy of the older and less precise Nanotrasen solution for non-lethal takedowns. This gun has smaller capacity in exchange for S-cells use."
 	icon = 'icons/obj/guns/energy/stunrevolver_moebius.dmi'
 	matter = list(MATERIAL_PLASTEEL = 12, MATERIAL_STEEL = 6, MATERIAL_SILVER = 2, MATERIAL_PLASTIC = 5)
+
+/obj/item/gun/energy/stunrevolver/relic
+	name = "NanoTrasen \"Zeus\" Prototype"
```

Comment:
```suggestion
	name = "NT SP \"Zeus\" Prototype"
```
NanoTrasen and NeoTheology intentionally have the same prefix in gun names.
Sure, this guns is a special snowflake, but "Prototype" in the name should be enough in my opinion.

Reply:
dongus